spec/factories/item.rb

```diff
@@ -4,7 +4,7 @@
   factory :item do
     name "EMA-10"
     description "Ford F-150"
-    source "Main Street Ford"
+    association :vendor, factory: :vendor
```

Comment:
I would prefer not to have the factory create associated objects. Doing this can lead to a cascade of creations. Any spec that needs it can create it, or stub it out.

Reply:
I will remove it